docs/content/languages/python.md

```diff
@@ -33,8 +33,8 @@ python-2.7.8
 
 ### Supported Runtimes
 
-The latest `python-2.7` and `python-3.6` are officially supported, but any
-runtime between 2.4.4–3.6.1 can be used, including PyPy runtimes. See
+The latest `python-2.7.x` and `python-3.6.x` are officially supported, but any
+runtime between 2.4.4–3.6.2 can be used, including PyPy runtimes. See
```

Comment:
This will continue to become stale over time, we should just reference the buildpacks [own documentation](https://github.com/heroku/heroku-buildpack-python#specify-a-python-runtime).

Reply:
I concur, how is the rewording in https://github.com/flynn/flynn/pull/4229/commits/fc1652b67267916b9ead2b7d2daed63716c59d43 ?